src/Common/ProgressIndication.cpp

```diff
@@ -1,22 +1,69 @@
-#include "ProgressBar.h"
+#include "ProgressIndication.h"
 #include <IO/WriteBufferFromFileDescriptor.h>
 #include <Common/TerminalSize.h>
 #include <Common/UnicodeBar.h>
 #include <Databases/DatabaseMemory.h>
 
+/// FIXME: progress bar in clickhouse-local needs to be cleared after query execution
```

Comment:
Should this be fixed before we merge?

Reply:
It should be done here https://github.com/ClickHouse/ClickHouse/pull/24535.